package.json

```diff
@@ -6,11 +6,10 @@
   "repository": "https://github.com/BrightspaceUI/testing.git",
   "scripts": {
     "lint": "eslint . --ext .js",
-    "start": "npx d2l-test-vdiff-report",
     "test": "npm run lint && npm run test:bin && npm run test:server && npm run test:browser",
     "test:bin": "mocha './test/bin/**/*.test.js'",
     "test:browser": "npm run test:browser:other && npm run test:browser:ctor",
-    "test:browser:other": "npx d2l-test-runner --files './test/browser/**/*.test.js'",
+    "test:browser:other": "npx d2l-test-runner --files ./test/browser/**/*.test.js",
```

Comment:
What was the issue with quotes for you? This can cause issues on POSIX system because it tries to pre-fill the wildcard.

Reply:
Error was:
```
Error: Could not find any test files with pattern(s): './test/browser/**/*.test.js',!**/node_modules/**/*
    at createTestSessions (C:\Desire2Learn\Instances\BrightspaceUI\testing\node_modules\@web\test-runner-core\dist\runner\createSessionGroups.js:55:19)
    at new TestRunner (C:\Desire2Learn\Instances\BrightspaceUI\testing\node_modules\@web\test-runner-core\dist\runner\TestRunner.js:32:115)
    at Object.startTestRunner [as start] (C:\Desire2Learn\Instances\BrightspaceUI\testing\node_modules\@web\test-runner\dist\startTestRunner.js:24:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async runTests (file:///C:/Desire2Learn/Instances/BrightspaceUI/testing/bin/d2l-test-runner.js:47:2)
```
Maybe we need to trim the quotes off in the client?

Reply:
Was that locally or in CI? I think I saw @dlockhart do this another way somewhere but I can't remember what it was. Escape the `*`, maybe?

Reply:
Locally. We ran into something similar here, but that was a totally different library. Seems odd.https://github.com/BrightspaceUI/core/pull/4222

Reply:
Hmm, not recalling "the other way".

Reply:
Looks like escaped quotes work. Still not sure what exactly changed, but I assume this is what could be used to get around the POSIX system issue?